tests/system/test_system.py

```diff
@@ -333,11 +332,17 @@ def test_update_document(client, cleanup):
         document.update({"bad": "time-past"}, option=option4)
 
     # 6. Call ``update()`` with invalid (in future) "last timestamp" option.
-    timestamp_pb = timestamp_pb2.Timestamp(
-        seconds=snapshot4.update_time.nanos + 3600, nanos=snapshot4.update_time.nanos
-    )
+    # TODO(microgen): start using custom datetime with nanos in protoplus?
+    # timestamp_pb = timestamp_pb2.Timestamp(
+    #     seconds=snapshot4.update_time.nanos + 3600, nanos=snapshot4.update_time.nanos
+    # )
+    timestamp_pb = _datetime_to_pb_timestamp(snapshot4.update_time)
+    timestamp_pb.seconds += 3600
```

Comment:
I'm confused about the significance of `3600` and why there seems to be a cast from nanos to seconds without conversion.

Reply:
deleted comment. That was weird, and the old code.